src/components/skeleton/Header/Header.js

```diff
@@ -28,7 +29,9 @@ export default class Header extends Component {
     const t = this.context.t;
     const loginOrProfile = !this.props.user.playerId ?
       <FlatButton label={t('nav.login')} onClick={() => browserHistory.push(t.route('login'))} /> :
-      <Link className="Header-link Header-icon-right" to={t.route('profile')}><Icon fa="fa fa-2x fa-user" /></Link>;
+      <Link className="Header-link Header-icon-right" to={t.route('profile')}>
+          <Icon fa="fa fa-2x fa-user" />&nbsp;&nbsp;<b>{this._reverseName(storeUtil.getPlayer(this.props.user.playerId).name)}</b>
+      </Link>;
```

Comment:
Elke persoon weet wie hij is... Dirk B kan niet inloggen in Dirk DS zijn account of omgekeerd.
Als ze paswoorden delen (waarom zou iemand dat doen?) dan moeten ze dat zelf maar onthouden...

nu gebruik je op mobile 30+% van de header (en mijn naam is dan nog niet volledig)

het is inderdaad zo dat je nergens je naam kan zien:
dat kan mss op de profiel pagina getoond worden?